test/app/features/help_center/data/repositories/guardian_repository_test.dart

```diff
@@ -170,18 +181,18 @@ void main() {
             status: 'pending',
           );
           final expected = right(const ValidField());
-          when(dataSource.delete(any))
+          when(() => dataSource.delete(guardian))
               .thenAnswer((_) async => const ValidField());
           // act
           final received = await sut.delete(guardian);
           // assert
-          expect(received, expected);
+          expect(expected, received);
```

Comment:
Os argumentos do `expect` estão invertidos, da maneira que esta caso o teste falhe a mensagem de erro será confusa, mostrando que o valor retornado como se fosse o correto (esperado).
```suggestion
          expect(received, expected);
```

Reply:
Ok, vou atualizar isto. Mas tenho dois pontos com relação a este ponto:
1. isto não foi alterado, não foi alvo do atualização do código. é o comportamento que já existe;
2. se isto é uma normativa e passará a ser exigido no Pull Request, então deveria ter uma documentação no README afirmando isto